README.md

```diff
@@ -14,6 +15,16 @@ Currently CSV reports are generated for WCAG2A and WCAG2AA standards using both
 - npm 6.14.8
 
 ## Installation
+1. clone the repo using:  
+`git clone git@github.com:ryanbuhl-lab49/python_pa11y_runner.git`
+2. change into the `python_pa11y_runner` directory  
+`cd python_pa11y_runner`  
+3. create a virtual environment:  
+`python -m venv .pa11y-runner-venv`
+4. activate the virtual environemnt
+`. .pa11y-runner-venv/bin/activate` on *nix platforms or  
+`.pa11y-runner-venv\Scripts\activate` on Windows platforms
+5. install the dependencies with `pip` using:
 `pip install -r requirements.txt`
 
 ## Usage
```

Comment:
It could be cool to include a little recording of the usage using this tool: https://asciinema.org/

Reply:
I've tried installing asciinema on my mac, but I'm not able to.  Homebrew complains during the install that Python 3.8 can't be installed as a binary and has to be built from source, but I'm not sure why as I already have Python 3.8 installed on my machine.  When I run the command that is suggested (`xcode-select --install`), that fails because the software "is not currently available from the Software Update server."

Reply:
Hmm OK, maybe we can come back to this later then.